tests/drivers/flash/src/main.c

```diff
@@ -140,12 +140,11 @@ static void test_read_unaligned_address(void)
 	}
 }
 
-void test_main(void)
+static void *flash_driver_setup(void)
 {
-	ztest_test_suite(flash_driver_test,
-		ztest_unit_test(test_setup),
-		ztest_unit_test(test_read_unaligned_address)
-	);
+	test_setup();
 
-	ztest_run_test_suite(flash_driver_test);
+	return NULL;
 }
```

Comment:
Why not just change the signature of `test_setup()`?

Reply:
Thanks for the review. Updated it.